CONTRIBUTING.md

```diff
@@ -54,16 +54,18 @@ contributions via GitHub pull requests. When contributing code please do the
 following:
 
 * Fork the repository and make your changes in a feature branch.
+* Please include new or update existing PlantUML sequence/architecture diagrams
+  that relate to your PR. ([/docs/diagrams/src/readme.md](details) on how we
+  maintain our diagrams)
 * Please include unit and integration tests for any new features and updates to
-existing tests.
+  existing tests.
 * Please ensure the unit and integration tests run successfully. Refer to the
-Testing section in the [README.md](README.md) on how to run these
-tests.
+  Testing section in the [README.md](README.md) on how to run these tests.
 * Please ensure that lint passes by running `./bin/run_lint`. The command should
```

Comment:
There appears to be multiple ways to run the linter, I think we should sit down as a group and decide on which command we should run. The two options are:
`./bin/run_lint`
or 
`./bin/build -l`

Reply:
how about we reference the test running instructions in `README.md` instead? That'll be easier to maintain

Reply:
Sure that's fine.